src/SIM/SIMoutput.h

```diff
@@ -314,6 +314,9 @@ class SIMoutput : public SIMinput
                          int& nBlock, std::vector< std::vector<int> >& sID,
                          size_t* i = nullptr);
 
+  //! \brief Private helper to write additional functions to VTF-file.
+  bool writeAddFuncs(int iStep, int& nBlock, int idBlock = 50, double time = 0.0);
```

Comment:
Since it is a private helper, it does not need default values either.

Reply:
indeed